pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.
```

Comment:
'Networking restrictions' needs some elaboration here, it makes no sense if you haven't read those other pages about what that means.   Also, one of the phrases below should be 'hoisted up' to this section as it applies to all clusters.

```suggestion
In order to attach an existing cluster to Kommander, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.

You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.  Networking restrictions refers to clusters that are located in a DMZ, behind a proxy server or a firewall, or that have additional requirements for access.  
```

Maybe we should also include links to the networking restrictions pages?

Reply:
do you mean this @cneth - 
/dkp/kommander/1.4/clusters/attach-cluster/cluster-with-networking-restrictions/

Reply:
Implemented as noted above, minus the lead-in "In order to attach...," and replaced with just "To attach...."